lms/djangoapps/student_account/urls.py

```diff
@@ -1,12 +1,14 @@
 from django.conf import settings
 from django.conf.urls import patterns, url
+from .views import RecoverPasswordView
 
 urlpatterns = []
 
 if settings.FEATURES.get('ENABLE_COMBINED_LOGIN_REGISTRATION'):
     urlpatterns += patterns(
         'student_account.views',
         url(r'^password$', 'password_change_request_handler', name='password_change_request'),
+        url(r'^recover-password$', RecoverPasswordView.as_view(), name="recover_password_api"),
```

Comment:
It turns out you do need a `{'post': 'post'}` for when this is a `ViewSet`. But now I must ask: any rationale for this being a `ViewSet` over a `View`, or simply a function similar to the one above it? This is why tests are currently failing.

Reply:
Because ```recover-password``` is an API endpoint so we should use Rest Framework. And I'm using ```ViewSet``` because on other parts of edX code uses ```ViewSet``` as well, so just to keep it similar.

Reply:
Grepping for `(View)` vs. `(ViewSet)` found me significantly more results for the former -- but okay, at this point we're beating the bush, so let's just go with this.